dspace/config/pages/add.xml

```diff
@@ -14,7 +14,7 @@
 <hi rend="bold">Submitting files</hi>
 <p>Submit files to:</p>
 <list>
-<item>University of Missouri--Columbia: <xref target="mailto:mospace@missouri.edu">mospace@missouri.edu</xref></item>
+ <item>University of Missouri--Columbia: Submit <xref target="library.missouri.edu/secure/mospace">online</xref> or via email to <xref target="mailto:mospace@missouri.edu">mospace@missouri.edu</xref></item>
```

Comment:
this target attribute needs to be a fully-qualified URL, specifically https://library.missouri.edu/secure/mospace